modules/jupyter-widget/webpack.config.js

```diff
@@ -15,12 +15,26 @@ const rules = [
 ];
 
 // Packages that shouldn't be bundled but loaded at runtime
-const externals = ['@jupyter-widgets/base'];
+
+function getExternals() {
+  const {peerDependencies = {}} = packageInfo;
+  const externals = {};
+  for (const depName in peerDependencies) {
```

Comment:
You can skip this since there are no `peerDependencies`.

Reply:
Removed